examples/ray_ddp_example.py

```diff
@@ -75,8 +75,7 @@ def train_mnist(config,
 def tune_mnist(data_dir,
                num_samples=10,
                num_epochs=10,
-               num_hosts=1,
-               num_slots=4,
+               num_workers=1,
```

Comment:
turns out num_workers wasn't being set in the example (weird global scoping)

Reply:
Oh good point- thanks for catching this!